specpred.py

```diff
@@ -0,0 +1,205 @@
+import sys
+from pyteomics import mzml, mass
+import numpy as np
+import pandas as pd
+
+#import spectrum_utils.spectrum as sus
+#import spectrum_utils.plot as sup
+import matplotlib.pyplot as plt
+
```

Comment:
This file seems to re-use a lot of code that is used in pltpred.py. Could you perhaps create a shared utils.py file?